code/src/components/MyThoughts.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react'
+import styled from 'styled-components';
+
+import { ContainerWrapper, FlexWrapper, SectionWrapper, SectionHeadings } from 'GlobalStyles';
+
+import { ARTICLE_URL } from 'data/Urls';
+import MyThoughtsPic from '../images/doSomethingGreat.jpg';
+
+// react component
+
+export const MyThoughts = () => {
+  return (
+    <SectionWrapper bgColor="#f8e9f0">
+      <ContainerWrapper>
+        <FlexWrapper>
+          <SectionHeadings>my thoughts</SectionHeadings>
+          <AllMyThoughtsCards>
+            <a
+              href={ARTICLE_URL}
+              target="_blank"
+              rel="noreferrer"
+              alt="bild">
+              <img src={MyThoughtsPic} lang="eng" alt="Medium article" />
+              <DateTitle>October 2022</DateTitle>
+              <ArticleTitle>Making a mid-life career change and everything that goes with it…
+              </ArticleTitle>
+              <ArticleText>
+                Some thoughts about changing my path.
+                Atend a coding BootCamp and for 22weeks,
+                not having any problems whatsoever knowing what you are doing
+                in your spare time<b> &gt;&gt;</b>
+              </ArticleText>
+            </a>
+          </AllMyThoughtsCards>
+        </FlexWrapper>
+      </ContainerWrapper>
+    </SectionWrapper>
+
```

Comment:
This is a great looking component and BIG thumbs up for the article and the confidence it took to publish! 👏